js/script.js

```diff
@@ -1,50 +1,79 @@
-let bookLib = JSON.parse(localStorage.getItem('bookLib')) || [];
-function getBook() {
-  const book = {};
-  book.title = document.querySelector('.title').value;
-  book.author = document.querySelector('.author').value;
-  book.id = '';
-
-  if (bookLib === null) {
-    book.id = 0;
-  } else {
-    book.id = bookLib.length;
+class Book {
+  id = 0;
+
+  title = '';
+
+  author = '';
+
+  constructor(id, title, author) {
+    this.id = id;
+    this.author = author;
+    this.title = title;
+  }
+}
+
+class bookLib {
+  bookList = [];
+
+  constructor(bookList) {
+    this.bookList = bookList;
+  }
+
+  addBook(book) {
+    this.bookList.push(book);
+    localStorage.setItem('bookLib', JSON.stringify(this.bookList));
+    UI(book);
   }
+
+  removeBook(book) {
+    const newBooks = [];
+    for (let i = 0; i < this.bookList.length; i++) {
+      if (this.bookList[i].id !== book.id) {
+        newBooks.push(this.bookList[i]);
+      }
+    }
+    this.bookList = newBooks;
+    localStorage.setItem('bookLib', JSON.stringify(this.bookList));
+  }
+}
+
+function getBook() {
+  const title = document.querySelector('.title');
+  const author = document.querySelector('.author');
+  const book = new Book(myBookLib.bookList.length + 1, title.value, author.value);
```

Comment:
- [ ] Please avoid setting the id to the array's length, as it will cause issue when removing the book using that Id, you should try to use a unique id , you can use [uuid](https://www.npmjs.com/package/uuid) if you prefer, as currently when I delete a book in the middle say the second one and add a new book both the book will have the same Id 3 and removing third will remove both of the book, I have recorded a small demo for your understanding please try to fix this 

https://user-images.githubusercontent.com/97521557/228316505-4df1f6b5-53b0-4e2c-aedc-a307858aeca9.mp4